tee-worker/app-libs/parentchain-interface/src/integritee/mod.rs

```diff
@@ -143,11 +143,7 @@ where
 			"[ShieldFundsAndInvokeFilter] attempting to execute indirect call with index {:?}",
 			index
 		);
-		if index == metadata.shield_funds_call_indexes().ok()? {
```

Comment:
I have a question here, So we are only skipping parsing `shield_funds`, is this sufficient?

Reply:
Since we don't have such extrinsic in pallet-teebag, we shouldn't land in this arm at all.